docs/microprofile.md

```
@@ -1,73 +1,97 @@
 # MicroProfile Integration
 
-The `hiero-enterprise-microprofile` module provides CDI-based integration for Eclipse MicroProfile runtimes such as Quarkus.
+The `hiero-enterprise-microprofile` module provides integration support for Eclipse MicroProfile-compatible applications such as Quarkus and Helidon.
+
+---
 
 ## Dependency
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)